### DISCUSSION TITLE
Geospatial data platform

### DISCUSSION THREAD

**Discussion by KhoulaDSD** [TDRA-ae/aegov-dls#10] [Q&A]
Are there any predefined templates for Geospatial data platform for us to refer. If you have any sample data, do share with us. It will be highly appreciated.

**Comment by moiz-conceptlz** [comment]
Hello @KhoulaDSD 

Please refer to the discussion [https://github.com/orgs/TDRA-ae/discussions/9](https://github.com/orgs/TDRA-ae/discussions/9) which discusses the same.
